### PR TITLE
feat(storage): add change default storage class sample

### PR DESCRIPTION
**Pull request by AlisskaPie**
Related to #1707

### CODE REVIEW

**Comment by AlisskaPie** [comment]
Waiting for [this PR](https://github.com/googleapis/google-cloud-go/pull/3038) to be merged first

File: storage/buckets/change_default_storage_class.go [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+	if err != nil {
+		return fmt.Errorf("storage.NewClient: %v", err)
+	}
+	defer client.Close()
+
+	ctx, cancel := context.WithTimeout(ctx, time.Second*10)
+	defer cancel()
+
+	bucket := client.Bucket(bucketName)
+	// See the StorageClass documentation for other valid storage classes:
```
Comment (tritone) [inline storage/buckets/change_default_storage_class.go:41]: This comment is not necessary; it's embedded in a page that contains this link and other languages don't have it.

**Review by tritone — COMMENTED** [review]
Needs an update to go.mod for CI to pass.

**Review by tbpg — APPROVED** [review]

**Comment by tritone** [comment]
Looks like there was a spanner integration test flake on go1.11. I'll re-run.

**Review by tritone — APPROVED** [review]

**Pull request merged**